fixtures/apidocs_test_case.py

```diff
@@ -9,16 +10,16 @@
 from sentry.testutils.cases import APITestCase
 from sentry.testutils.helpers.datetime import before_now, iso_format
 from sentry.testutils.skips import requires_snuba
-from sentry.utils import json
 
 
 @requires_snuba
 class APIDocsTestCase(APITestCase):
     @functools.cached_property
     def cached_schema(self):
         path = os.path.join(os.path.dirname(__file__), "../tests/apidocs/openapi-derefed.json")
-        with open(path) as json_file:
-            data = json.load(json_file)
+        # opens the file as binary
```

Comment:
can probably nuke the comment since `'rb` is fairly explanatory

Reply:
Removed them all